src/components/Feed/Feed.js

```diff
@@ -17,6 +36,7 @@ const Feed = ({ component: El, data, dateKey }) => {
       <Fragment key={id}>
         {isNotSameAsLastDate && <h4 className={styles.date}>{date}</h4>}
         <El className={styles.signal} {...item} />
+        {index === showAfterElNumber && <FeaturedInsightsBlock />}
```

Comment:
The main flaw in the current logic is that `Featured insights` view change between desktop/mobile devices happens via css. That's a lot of additional calls + memory garbage that React should handle and reconcile => big chunk of unnecessary weight for mobile devices.

**How we can solve it?**
There should be only one `InsightsFeaturedBlock` in two different places (at the side and in the middle of the feed), that should be conditionally rendered via `isDesktop` prop

Reply:
isDesktop prop is for screen > than 768. For this case it isn't suitable, because on < 992px it should be in another place. Or I should create new var, that check width <= 992?

Reply:
Oh, I see. Sure, create a new var for now